frontend/src/constants/CreateAssessmentConstants.ts

```diff
@@ -1,40 +1,8 @@
-const gradeOptions = [
-  {
-    label: "K",
-    value: "k",
-  },
-  {
-    label: "Grade 1",
-    value: "1",
-  },
-  {
-    label: "Grade 2",
-    value: "2",
-  },
-  {
-    label: "Grade 3",
-    value: "3",
-  },
-  {
-    label: "Grade 4",
-    value: "4",
-  },
-  {
-    label: "Grade 5",
-    value: "5",
-  },
-  {
-    label: "Grade 6",
-    value: "6",
-  },
-  {
-    label: "Grade 7",
-    value: "7",
-  },
-  {
-    label: "Grade 8",
-    value: "8",
-  },
-];
+import { Grade } from "../APIClients/types/UserClientTypes";
+
+const gradeOptions = Object.keys(Grade).map((grade) => ({
+  value: grade,
+  label: grade,
```

Comment:
I now noticed I forgot to add titlecase to this label, will do so with any additional review changes in the next commit